package.json

```diff
@@ -23,25 +23,25 @@
   "peerDependencies": {
     "@emotion/react": "^11.1.5",
     "@guardian/libs": "^3.1.0",
-    "@guardian/source-foundations": "^4.2.1",
-    "@guardian/source-react-components": "^4.4.0",
+    "@guardian/source-foundations": "^7",
+    "@guardian/source-react-components": "^9",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "typescript": "^4.1.3"
   },
   "devDependencies": {
-    "@babel/core": "^7.12.10",
-    "@babel/plugin-transform-runtime": "^7.9.0",
-    "@babel/preset-env": "^7.5.5",
-    "@babel/preset-react": "^7.0.0",
-    "@babel/preset-typescript": "^7.12.7",
+    "@babel/core": "^7.19.6",
+    "@babel/plugin-transform-runtime": "^7.19.6",
+    "@babel/preset-env": "^7.19.4",
+    "@babel/preset-react": "^7.18.6",
+    "@babel/preset-typescript": "^7.18.6",
     "@changesets/cli": "^2.24.1",
     "@emotion/babel-preset-css-prop": "^10.0.14",
     "@emotion/react": "^11.1.5",
-    "@guardian/libs": "^3.1.0",
+    "@guardian/libs": "^9.0.1",
```

Comment:
We already have a PR for this: https://github.com/guardian/dotcom-rendering/pull/6174